src/components/AddressForm.tsx

```diff
@@ -126,15 +155,16 @@ function AddressForm({city, country, formID, onAddressChanged, onSubmit, shouldS
         >
             <View>
                 <InputWrapper
+                    // @ts-expect-error TODO: Remove this once InputWrapper (https://github.com/Expensify/App/issues/25109) is migrated to TypeScript.
                     InputComponent={AddressSearch}
                     inputID="addressLine1"
                     label={translate('common.addressLine', {lineNumber: 1})}
-                    onValueChange={(data, key) => {
+                    onValueChange={(data: string, key: string) => {
                         onAddressChanged(data, key);
                         // This enforces the country selector to use the country from address instead of the country from URL
-                        Navigation.setParams({country: undefined});
+                        Navigation.setParams({country: undefined}, '');
                     }}
-                    defaultValue={street1 || ''}
+                    defaultValue={street1 ?? ''}
```

Comment:
street1 has default value `''`, it will cause a regression
```suggestion
                    defaultValue={street1 || ''}
```

Reply:
Why do you think we can have a regression here? 

For me looks like there is no regression, but it is safe just to remove these statements since they already have default values

Reply:
Oh, you are right, let's remove it then

Reply:
removed